### DISCUSSION TITLE
Adding BaseOperatorLink to mapped tasks

### DISCUSSION THREAD

**Discussion by tagoad** [apache/airflow#27631] [Q&A]
Does adding Extra Links to mapped tasks not work?

**Comment by potiuk** [comment]
Does it work?

**Reply by tagoad** [reply]
Doesn't seem to be, but i wasn't sure if there was additional configuration needed for mapped tasks or not.

**Reply by potiuk** [reply]
I also do not know :). But if it does not work - then I think you should start with explaining what does not work and what you are trying to do - and showing logs and code.

**Reply by tagoad** [reply]
I figured it was a simple enough question that all that wasn't necessary. Either the feature is implemented or it isn't. If i thought it was a bug or something i would have made a issue report about it with all that.

**Reply by potiuk** [reply]
Well, it is not simple if you do not explin what you want to do and where you think it failed. The question is simple but makes not a lot of sense. Adding a BaseOperatorLink to mapped task does not make to much sense. You might add BaseOperatorLink to a custom operator - but at this stage it is not even known whether it will be used as mapped task (any operator can be used this way). If you have custom operator with Operator Link added it should work regardless.

So I stll will suggest that you explain what you tried to do and where it failed if you think it's a bug.
